scripts/extensions/sams/src/components/sets/setEditorPanel.tsx

```diff
@@ -251,13 +251,13 @@ export class SetEditorPanelComponent extends React.Component<IProps, IState> {
                                 </FormRow>
                                 <FormRow>
                                     <Select
-                                        label={gettext('Stoeage Unit')}
+                                        label={gettext('Storage Unit')}
                                         onChange={this.onChange.storage_unit}
                                     >
-                                        <Option>{gettext('Bytes')}</Option>
-                                        <Option>{gettext('KB')}</Option>
-                                        <Option>{gettext('MB')}</Option>
-                                        <Option>{gettext('GB')}</Option>
+                                        <Option>{DATA_UNIT.BYTES}</Option>
```

Comment:
To provide translations here, the `DATA_UNIT.*` values will need to be applied as the `value` attribute of the tag, and the innerHTML use `gettext`. i.e.
`<Option value={DATA_UNIT.BYTES}>{gettext('Bytes')}</Option>`

Reply:
Okay.